content/en/user-guide/tools/aws-replicator/index.md

```diff
@@ -0,0 +1,310 @@
+
+## Introduction
+
+LocalStack AWS Replicator copies AWS resources into a running LocalStack instance.
+It helps when deploying applications that rely on existing resources like SSM parameters or VPCs, ensuring they are available before deploying the main stack.
+
+This removes the need to change existing stacks or create custom infrastructure, making LocalStack setup easier.
+
+{{< callout "note">}}
+The AWS Replicator is in a preview state, supporting only [selected resources](#supported-resources).
+It is only available as part of the **LocalStack Teams** plan and higher.
+{{< /callout >}}
+
+## Getting started
+
+A valid `LOCALSTACK_AUTH_TOKEN` must be configured to start the LocalStack Pro image.
+
+{{< callout "note" >}}
+The Replicator is in limited preview and is available from LocalStack CLI version 4.2.0.
+If you encounter issues, update your [LocalStack CLI](https://docs.localstack.cloud/getting-started/installation/#updating).
+{{< /callout >}}
+
+### Retrieve credentials to access AWS
+
+The AWS Replicator needs read access to your AWS account and can perform a limited set of read-only operations on supported resources.
+
+Replication is triggered using the LocalStack CLI, which must run in a shell configured to access AWS.
+If you have the aws-cli v2 installed, the cli will read credentials from your configured `AWS_PROFILE`.
+
+Otherwise, the following environment variables must be set:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN` (optional)
+- `AWS_DEFAULT_REGION`
+
+{{< callout "tip" >}}
+Use `aws configure export-credentials --format env` to print the required environment variables in a format that can be evaluated.
+
+{{< command >}}
+<disable-copy>$ </disable-copy>eval $(AWS_PROFILE=<aws-profile> aws configure export-credentials \
+ --format env)
+{{< /command >}}
+{{< /callout >}}
+
+### Trigger a replication job
+
+Replication jobs can be triggered using the LocalStack CLI or an HTTP API.
+Both methods have two steps:
+
+1. Submit a replication job.
+2. Check the job status.
+
+#### Using the LocalStack CLI
+
+The Replicator CLI is part of the LocalStack CLI.
+Follow the [installation instructions](https://docs.localstack.cloud/getting-started/installation/#localstack-cli) to set it up.
+
+To start a replication job, get the ARN of the resource to replicate.
+Then, trigger the job using the command:
+
+{{< command >}}
+export LOCALSTACK_ENABLE_REPLICATOR=1
+export LOCALSTACK_AUTH_TOKEN=<auth token>
+export AWS_DEFAULT_REGION=...
+<disable-copy>
+# if required
+# export AWS_ACCESS_KEY_ID=
+# export AWS_SECRET_ACCESS_KEY=
+</disable-copy>
+localstack replicator start \
+ --resource-type <resource-type> \
+ --identifier <identifier> \
+ [--target-account-id <account-id>] \
+ [--target-region-name <region-name>]
+{{< /command >}}
+
+{{< callout "note" >}}
+Resources that supports replicating with arn can be replicated by providing `--resource-arn` instead of `--resource-type` and `--identifier`.
```

Comment:
```suggestion
Resources that supports replicating with arn can be replicated by providing `--resource-arn` instead of `--resource-type` and `--resource-identifier`.
```